src/styles/colors.scss

```diff
@@ -0,0 +1,5 @@
+$white: #FFF;
+$light-white: #F3F3F3;
+$dark-blue: #060B35;
+$gray: #616070;
+$light-blue: #00ACDC;
```

Comment:
Variable names should show their purpose, because if the designer changes the color to red it will be strange that `$light-blue` actually red. Example `$main-color`